### PR TITLE
LPS-147852 Add test OrderButtonAsAToggleCanDisplayTooltip

### PR DESCRIPTION
**Pull request by susanchen3**
**Background**
Create automated tests for Management Toolbar.

**Test Plan**
Given a user of the portal
When the user opens any section with the management toolbar
And clicks on the order button
Then there is no dropdown menu
And the button is changed to the other order direction
And a tooltip message will be displayed

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-147852

### CODE REVIEW

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/ManagementToolbar.testcase [hunk 1]
```diff
@@ -103,6 +103,47 @@ definition {
...
+		}
+
+		task ("Then there is no dropdown menu") {
+			AssertElementNotPresent(
+				key_item = "",
+				locator1 = "ManagementBar#ACTIVE_DROPDOWN_ITEM");
+		}
+
+		task ("And the button is changed to the other order direction") {
+			AssertElementPresent(locator1 = "xpath=(//nav[contains(@class,'management-bar')]//*[*[name()='svg'][contains(@class,'icon-order-list-down')] and contains(@title,'Reverse Order Direction')])");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/ManagementToolbar.testcase:131]: Why not create a poshi path for this?

**Review by john-co — APPROVED** [review]
Besides the question, looks good to me :sunglasses:

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/ManagementToolbar.testcase (hunk 1 above)
Comment (susanchen3) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/ManagementToolbar.testcase:131]: oh.. right... will create one :smile:

**Comment by susanchen3** [comment]
ci:test:sf

**Comment by susanchen3** [comment]
ci:forward

**Pull request closed without merging**